batchflow/models/eager_torch/base.py

```diff
@@ -0,0 +1,1203 @@
+""" Eager version of TorchModel. """
+import os
+import re
+import threading
+import inspect
+from collections import OrderedDict
+from functools import partial
+from pprint import pprint
+
+import numpy as np
+import torch
+import torch.nn as nn
+from torch.utils.tensorboard import SummaryWriter
+
+from .utils import unpack_fn_from_config, get_shape
+from .layers import ConvBlock
+from .losses import CrossEntropyLoss
+from ... import Config
+
+
+
+LOSSES = {
+    'mse': nn.MSELoss,
+    'bce': nn.BCEWithLogitsLoss,
+    'ce': CrossEntropyLoss,
+    'crossentropy': CrossEntropyLoss,
+    'absolutedifference': nn.L1Loss,
+    'l1': nn.L1Loss,
+    'cosine': nn.CosineSimilarity,
+    'cos': nn.CosineSimilarity,
+    'hinge': nn.HingeEmbeddingLoss,
+    'huber': nn.SmoothL1Loss,
+    'logloss': CrossEntropyLoss,
+}
+
+DECAYS = {
+    'exp': torch.optim.lr_scheduler.ExponentialLR,
+    'lambda': torch.optim.lr_scheduler.LambdaLR,
+    'step': torch.optim.lr_scheduler.StepLR,
+    'multistep': torch.optim.lr_scheduler.MultiStepLR,
+    'cos': torch.optim.lr_scheduler.CosineAnnealingLR,
+}
+
+DECAYS_DEFAULTS = {
+    torch.optim.lr_scheduler.ExponentialLR : dict(gamma=0.96),
+    torch.optim.lr_scheduler.LambdaLR : dict(lr_lambda=lambda epoch: 0.96**epoch),
+    torch.optim.lr_scheduler.StepLR: dict(step_size=30),
+    torch.optim.lr_scheduler.MultiStepLR: dict(milestones=[30, 80]),
+    torch.optim.lr_scheduler.CosineAnnealingLR: dict(T_max=None)
+}
+
+
+
+class EagerTorch:
+    r""" Base class for eager Torch models.
+
+    Parameters
```

Comment:
I just thought that what is listed below are not actually parameters, but keys of config. What about writing about it explicitly?